gui/default/index.html

```diff
@@ -768,6 +768,13 @@ <h4 class="panel-title">
                       <th><span class="fas fa-fw fa-folder"></span>&nbsp;<span translate>Folders</span></th>
                       <td class="text-right" ng-attr-title="{{deviceFolders(deviceCfg).map(folderLabel).join(', ')}}">{{deviceFolders(deviceCfg).map(folderLabel).join(", ")}}</td>
                     </tr>
+                    <tr ng-if="findDevice(deviceCfg.deviceID).remoteGUIPort !== '0'">
+                      <th><span class="fas fa-fw fa-desktop"></span>&nbsp;<span translate>Web Address</span></th>
```

Comment:
In some parts it's referred to as `Web Address` in others as "Remote GUI` etc, I think the naming of strings and variables should be normalised across the board.

Reply:
done

Reply:
It's still web address here, remote gui address in the variables.
I think the variables could be idtoWebAddress? webAddressCache, etc?